utils/splunk/splunk_helper.py

```diff
@@ -99,16 +99,16 @@ def dispatch(self, saved_search, splunk_user, splunk_app, parameters):
         :return: the sid of the saved search
         """
         dispatch_path = '/servicesNS/%s/%s/saved/searches/%s/dispatch' % (splunk_user, splunk_app, quote(saved_search.name))
-        response_body = self._do_post(dispatch_path, parameters, saved_search.request_timeout_seconds).json()
-        return response_body["sid"]
+        response_body = self._do_post(dispatch_path, parameters, saved_search.request_timeout_seconds, splunk_ignore_config).json()
+        return response_body.get("sid")
 
     def _do_get(self, path, request_timeout_seconds, verify_ssl_certificate):
         url = "%s%s" % (self.instance_config.base_url, path)
         response = self.requests_session.get(url, timeout=request_timeout_seconds, verify=verify_ssl_certificate)
         response.raise_for_status()
         return response
 
-    def _do_post(self, path, payload, request_timeout_seconds):
+    def _do_post(self, path, payload, request_timeout_seconds, splunk_ignore_config='true'):
```

Comment:
I interpret the name `splunk_ignore_config` that there is a specific config that ignores something(s). However, it is only a single true/false variable. Can we rename the variable to something more similar to what the user setting is, `ignore_saved_search_errors`?

Reply:
Done.